packages/web-twig/src/Resources/components/TextField/__tests__/__fixtures__/textFieldPasswordToggle.twig

```diff
@@ -4,3 +4,11 @@
     label="Text field with password toggle"
     name="passwordToggle"
 />
+
+<TextField
+    hasPasswordToggle
+    id="password-toggle"
+    label="Text field with password toggle"
+    name="passwordToggle"
+    isDisabled
+/>
```

Comment:
please add empty line to the end of the file

Reply:
new line added